src/components/MAPI/clusters/CreateCluster/index.tsx

```diff
@@ -287,35 +293,51 @@ const CreateCluster: React.FC<ICreateClusterProps> = (props) => {
               controlPlaneNode={state.controlPlaneNode}
               onChange={handleChange(ClusterPropertyField.Name)}
             />
-            <WorkerNodesCreateClusterDescription
+            <CreateClusterDescription
               id={`cluster-${ClusterPropertyField.Description}`}
               cluster={state.cluster}
               providerCluster={state.providerCluster}
               controlPlaneNode={state.controlPlaneNode}
               onChange={handleChange(ClusterPropertyField.Description)}
               autoFocus={true}
             />
-            <Box direction='row' margin={{ top: 'medium' }}>
-              <Button
-                bsStyle='primary'
-                disabled={!isValid}
-                type='submit'
-                loading={state.isCreating}
-              >
-                Create cluster
-              </Button>
-
-              {!state.isCreating && (
-                <Button bsStyle='default' onClick={handleCancel}>
-                  Cancel
-                </Button>
-              )}
-            </Box>
+            <CreateClusterRelease
+              id={`cluster-${ClusterPropertyField.Release}`}
+              cluster={state.cluster}
+              providerCluster={state.providerCluster}
+              controlPlaneNode={state.controlPlaneNode}
+              onChange={handleChange(ClusterPropertyField.Release)}
+            />
+            <CreateClusterControlPlaneNodeAZs
+              id={`cluster-${ClusterPropertyField.ControlPlaneNodeAZs}`}
+              cluster={state.cluster}
+              providerCluster={state.providerCluster}
+              controlPlaneNode={state.controlPlaneNode}
+              onChange={handleChange(ClusterPropertyField.ControlPlaneNodeAZs)}
+            />
             <Box margin={{ top: 'medium' }}>
-              <Text color='text-weak'>
-                Note that it takes around 30 minutes on average until a new
-                cluster is fully available.
-              </Text>
+              <Box direction='row'>
+                <Button
+                  bsStyle='primary'
+                  disabled={!isValid}
+                  type='submit'
+                  loading={state.isCreating}
+                >
+                  Create cluster
+                </Button>
+
+                {!state.isCreating && (
+                  <Button bsStyle='default' onClick={handleCancel}>
+                    Cancel
+                  </Button>
+                )}
+              </Box>
+              <Box margin={{ top: 'medium' }}>
+                <Text color='text-weak'>
+                  Note that it takes around 30 minutes on average until a new
+                  cluster is fully available.
```

Comment:
Since we are now creating a cluster without node pools in this form, I'm currently checking how long it takes until the k8s API is up when not launching worker nodes. Result pending...

Reply:
It took 12 minutes in my test. Lets change the text to:

`It will take around 15 minutes for the control plane to become available.`